tests/src/server/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}
 # Tests:
 
 set(TESTS
+  test_qgsserver_interface.cpp
```

Comment:
```suggestion
  testqgsserverinterface.cpp
```

I think the naming convention for cpp tests has no `_`

Reply:
Indeed you're right :+1: